### PR TITLE
updating GA

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/OfficeDocs-SharePoint/issues/3123

@v-smandalika please review

image

### CODE REVIEW

**Comment by get-itips** [comment]
@scanum

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@v-smandalika PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by get-itips** [comment]
@tiburd Tina, can you help me getting this merged? Thank you

File: SharePoint/SharePointOnline/data-access-governance-reports.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ description: "In this article, you'll learn about reports that can help you gove
 As security and compliance requirements increase across industries and the amount of business data grows exponentially, organizations need tools to help govern their data. Data access governance reports provide info that helps you govern access to SharePoint data. The reports help you discover sites that contain potentially overshared or sensitive content. You can use these reports to assess and apply appropriate security and compliance policies.
 
 > [!NOTE]
-> This feature is in preview for organizations that have Microsoft 365 E5. We will begin rolling out in mid-November and be complete by mid-December.
+> This feature is in preview for organizations that have Microsoft 365 E5. We will begin rolling out in mid-February (previously mid-January) and be complete by mid-March (previously mid-February).
```
Comment (tiburd) [inline SharePoint/SharePointOnline/data-access-governance-reports.md:28]:
```suggestion
> At the time of this writing, this feature is in preview for organizations that have Microsoft 365 E5. It is scheduled to be rolled out in the near future. 
```

**Comment by tiburd** [comment]
Changed text to not include actual dates.

**Pull request merged**